samples/containers/hello-back-button/src/main/java/com/squareup/sample/hellobackbutton/AreYouSureWorkflow.kt

```diff
@@ -75,7 +64,7 @@ object AreYouSureWorkflow : StatefulWorkflow<Unit, State, Finished, AlertContain
         )
       }
       Quitting -> {
-        val dialog = AlertScreen(
+        val dialog = AlertModalRendering(
```

Comment:
This is the punchline. `AlertModalRendering` is bound in the `ViewRegistry` to `AlertDialogBuilder`. We didn't have to make a subclass of `ModalContainerView` in order to show this particular kind of modal.